features-json/picture-in-picture.json

```diff
@@ -329,16 +337,19 @@
   },
   "notes":"",
   "notes_by_num":{
-    "1":"Apple provides an equivalent proprietary API for Picture-in-Picture."
+    "1":"Apple provides an equivalent proprietary API for Picture-in-Picture.",
+	"2":"Enabled in Chrome through the \"#enable-picture-in-picture\" and \"#enable-surfaces-for-videos\" flag in chrome://flags",
```

Comment:
I'm not sure what is the value of this note. There were some Picture-in-Picture bugs in Chrome 69. These flags were temporary.

Reply:
It's just an additionnal note for Chrome 69. This kind of information are removed when the feature is published in the browser without flag?

Reply:
Picture-in-Picture is now working without any flag in Chrome 70. Flags will be removed soon.

Reply:
So we remove the information for Chrome 69?

Reply:
I think so yes.